html/components/button.stories.js

```diff
@@ -23,11 +23,24 @@ always use a \`<button>\` element.
 must include a \`title=””\` attribute.
 - If a Button only includes an Icon with no text,
 alternative text must be provided.
-- If the Button has a spinner setup, the \`aria-label\`
-will be set to 'Loading' when the button is
-clicked unless you supply a custom
-string value to the \`data-sprk-spinner-aria-label\`
-attribute.
+
+##### Spinner State
+- Spinner state requires additional engineering.
```

Comment:
I would elaborate what / why need additional engineering. "Spinner state requires additional engineering." doesn't jump out to me as being clear as to what needs to be done.

Reply:
I would mention that `cancelSpinning` sets the contents of the button back to an empty string so part of the additional engineering includes manually saving and restoring the button text if it is needed.

Reply:
the `data-sprk-spinner-text` attribute is actually able to restore the text! Just added it to the docs, thanks for reminding! Take a look at screenshot below to see if those changes accomplish your concerns.